circle.yml

```diff
@@ -0,0 +1,34 @@
+machine:
+  environment:
+    COVALENCE_CMD: "docker run -v /home/ubuntu/.aws:/root/.aws -v /var/run/docker.sock:/var/run/docker.sock -v /home/ubuntu/dockerfile-covalence/uat:/home/ubuntu/dockerfile-covalence/uat -w /home/ubuntu/dockerfile-covalence/uat"
```

Comment:
hrmm interesting. I'll fiddle with it later then. Not a blocker to use the file for now.